lib/open_uri.ml

```diff
@@ -17,12 +17,22 @@
 
 open Xapi_stdext_pervasives.Pervasiveext
 
+module D = Debug.Make (struct let name = "open_uri" end)
+open D
+
```

Comment:
Should I remove this module and do a `failwith` instead?

Reply:
The module is being used to log the error. I wouldn't remove that seeing what a pain it was to find where the `Not_Found` was being generated yesterday.

Reply:
I agree. :)